src/main/java/uk/gov/companieshouse/efs/api/submissions/controller/NewSubmissionController.java

```diff
@@ -13,9 +13,9 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import uk.gov.companieshouse.api.model.efs.submissions.PresenterApi;
 import uk.gov.companieshouse.api.model.efs.submissions.SubmissionResponseApi;
 import uk.gov.companieshouse.efs.api.submissions.service.SubmissionService;
+import uk.gov.companieshouse.efs.api.submissions.model.PresenterApi;
```

Comment:
There's a possibility that Spring Boot bean validation isn't getting applied to the method
`newSubmission(@RequestBody @Valid @NotNull PresenterApi presenterApi, ...`
despite the `@Valid` annotation on parameter `presenterApi`

As an alternative I'd like to spike replacing these changes to this file with an `@Validated` annotation at the class level.

i.e.

```
@RestController
@Validated
@RequestMapping("/efs-submission-api/submissions/new")
public class NewSubmissionController {
```

Reply:
Hi @hepsimo - we can give it a try. I will make that change here.